autopts/ptsprojects/mynewt/gap_wid.py

```diff
@@ -76,12 +72,33 @@ def hdl_wid_204(_: WIDParams):
     return btp.check_discov_results(addr_type=0x02)
 
 
+def hdl_wid_227(params: WIDParams):
+    bd_addr = btp.pts_addr_get()
+    bd_addr_type = btp.pts_addr_type_get()
+    stack = get_stack()
+
+    handle = btp.parse_handle_description(params.description)
+    if not handle:
+        return False
+
+    btp.gatt_cl_write(bd_addr_type, bd_addr, handle, '02', 1)
+    stack.gatt_cl.wait_for_write_rsp()
+
+    if (btp.verify_att_error("authentication error")):
+        btp.gap_pair()
+
+    if (btp.verify_att_error("insufficient encryption")):
+        btp.gap_pair()
+
+    return True
+
+
 def hdl_wid_242(_: WIDParams):
```

Comment:
hmm why there is no generic wid handler for this wid?

Reply:
I'll move this to general, I think that maybe some IXIT value makes Zephyr path different?